From 6b7a4522a36c45719582cb0003250be9f765d697 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Wed, 21 Apr 2021 16:07:56 +0200
Subject: [PATCH] Don't wrap dconf.h in a C block

Recent glib versions include some template stuff when C++ is being used,
which breaks this

There was a MR upstream to "fix" this but it was rejected
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1935
---
 src/mgconfitem.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/mgconfitem.cpp b/src/mgconfitem.cpp
index 8a8fc29..69fa5b8 100644
--- a/src/mgconfitem.cpp
+++ b/src/mgconfitem.cpp
@@ -17,9 +17,7 @@
 ****************************************************************************/
 
 // This has to be the first include otherwise gdbusintrospection.h causes an error.
-extern "C" {
 #include <dconf.h>
-};
 
 #include <QCoreApplication>
 #include <QString>
-- 
2.31.1

